submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,71 @@
+#############################################################################
+##Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/base/apache-hive-${HIVE_VERSION}-bin"
+
+# Set the port for Hive to listen on
+
+export HIVE_PORT="11112"
+
+# Set HADOOP_HOME (IF NOT DONE ABOVE)
+# Hive uses Hadoop, so:
+#    you must have Hadoop in your path OR
+#    export HADOOP_HOME=<hadoop-install-dir>
+#
+# export HADOOP_HOME="${_PREFIX}/base/hadoop-${HADOOP_VERSION}"
+
+# need ANT_HOME configured for Hive web interface
+export ANT_LIB="HIVEDIRPREFIX/base/apache-ant-1.9.9/lib"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option.  See README for more details.
+
+export HIVE_LOCAL_DIR="LOCALDIRPREFIX/hive"
+export HIVE_METASTORE_DIR="LUSTREDIRPREFIX/hive/database/postgres/data"
```

Comment:
HIVE_METASTORE_DIR is the location where Hive looks for the metastore
HIVE_DEFAULT_METASTORE_DIR is where it will go if HIVE_METASTORE_DIR does not exist

Reply:
Ahh, by "some explanation" I was thinking some comments should be added :-)

Are these environment variables used by hive, or are they your creation for Magpie?  I don't think it's necessary to have HIVE_DEFAULT_METASTORE_DIR.  if the directory the user lists for HIVE_METASTORE_DIR and it doesn't exist, Magpie should just create the directory.